### DISCUSSION TITLE
Connecting to a Firebird databases

### DISCUSSION THREAD

**Discussion by LeonOtto01** [osalvador/ReplicaDB#141] [Show and tell]
Hello everyone,
I just wanted to briefly share my experience connecting to a Firebird database and migrating data to PostgreSQL. Maybe some are as desperate as I am for such a solution, so here I have it.

I downloaded the official Firebird JDBC driver in version ([5.0.2](https://github.com/FirebirdSQL/jaybird/releases/download/v5.0.2/jaybird-5.0.2.java11.zip)). I copied the "jaybird-5.0.2.java11" file to /lib/ in my ReplicaDB folder. I then created a new config.

source.connect=jdbc:firebirdsql://localhost:3050/c:/YOUR_PATH/DEMO-DB.FDB
source.user=_username_
source.password=_password_
source.table=_database_
source.connect.parameter.driver=org.firebirdsql.jdbc.FBDriver

Now you just need to change the IP, path, username, password and database name of your Firebird instance.

I tested this on Windows 11 with Java 19 and the newest Firebird 3.0

**Comment by osalvador** [comment]
Hi @LeonOtto01 

What was your problem? Can you share the ReplicaDB execution log? 

Some functions still don't work Java > 8 :( 

Thank you!

**Reply by LeonOtto01** [reply]
Hey @osalvador 

There wasn't a real "problem". I just had to tinker a little before I figured out how to execute it correctly. I thought it could help some people find the correct solution even faster. You could maybe even implement it nativly and mention it in the compatible list.

Which functions do not work with Java > 8?
